translib/db/db.go

```diff
@@ -1148,27 +1188,22 @@ func (d *DB) StartTx(w []WatchKeys, tss []*TableSpec) error {
 	}
 
 	if len(args) == 1 {
-		glog.Warning("StartTx: Empty WatchKeys. Skipping WATCH")
-		goto StartTxSkipWatch
+		glog.Warning("performWatch: Empty WatchKeys. Skipping WATCH")
+		goto SkipWatch
```

Comment:
Can we have a style requirement to avoid goto ?
Please avoid here.

Reply:
Renuka, We have used goto throughout the code. Even in the checked in code we have goto. May I know the reason to avoid goto. I think we need to leave this to programmers preference.